acceptance_tests/features/environment.py

```diff
@@ -1,31 +1,43 @@
 import base64
 import json
+import shutil
 import uuid
 from datetime import datetime
+from pathlib import Path
 
 from acceptance_tests.utilities.rabbit_context import (
     RabbitContext
 )
 from config import Config
 
 
-def before_all(context):
+def before_all(_context):
```

Comment:
If a param is not used, the convention is to use a single `_` in its place (although I do appreciate the retention of the parameter's name):

```suggestion
def before_all(_):
```

If this is just to fix some IDE warnings then I'd suggest we stick to the full name (`context`) instead of coding to the tool.